### PR TITLE
ignore false positive link

### PR DESCRIPTION
**Pull request by karthik2804**
fix false positive link which is leading to CI to be red.

### CODE REVIEW

**Comment by itowlson** [comment]
I notice when I curl the URL, it gives a HTTP/2 response - I wonder if that's confusing the link checker.  Otherwise weird that it would do this!

**Comment by karthik2804** [comment]
maybe this was not needed? opened based on #1377

File: .build/check-broken-links.sh [hunk 1]
```diff
@@ -106,6 +106,7 @@ blc --recursive http://127.0.0.1:3000
     --exclude 'http://localhost:16686/'                                                                                                                                                     \
     --exclude 'http://localhost:5050/explore'                                                                                                                                               \
     --exclude 'http://localhost:5050/explore'                                                                                                                                               \
+    --exclude ' https://stackshare.io/stackups/powershell-vs-webassembly'                                                                                                                   \
```
Comment (itowlson) [inline .build/check-broken-links.sh:109]:
```suggestion
    --exclude 'https://stackshare.io/stackups/powershell-vs-webassembly'                                                                                                                   \
```

**Review by itowlson — APPROVED** [review]
LGTM, one teeny tiny formatting suggestion

**Comment by itowlson** [comment]
Maybe it objects to certain user agents or something.  Anyway happy to have it added

**Pull request merged**